noxfile.py

```diff
@@ -643,7 +643,7 @@ def _run_cargo_test(
     command = ["cargo"]
     if "careful" in session.posargs:
         command.append("careful")
-    command.append("test")
+    command += ("test", "--no-fail-fast")
```

Comment:
I know this is a super nit, but I think `.extend` would be more consistent with the usage of `.append` everywhere else in this function.